### ISSUE TITLE
[Bug]: failed to deserialize: missing field `mail`

### ISSUE THREAD

**Issue by pbek** [atuinsh/atuin#2404]
### What did you expect to happen?

@ellie, @tessus

The field `mail` from https://github.com/atuinsh/atuin/pull/2190 was not generated automatically in `server.toml`.
And I didn't find anything in https://docs.atuin.sh/self-hosting/server-setup/#configuration.
Is there some documentation what is needed?

### What happened?

When I run the latest docker image from https://github.com/atuinsh/atuin/pkgs/container/atuin (6e15286-amd64) for the first time, I get a:

     failed to deserialize: missing field `mail`

### Atuin doctor output

```yaml
atuin-server-1  | Error: could not load server settings
atuin-server-1  |
atuin-server-1  | Caused by:
atuin-server-1  |     failed to deserialize: missing field `mail`
atuin-server-1  |
atuin-server-1  | Location:
atuin-server-1  |     /app/crates/atuin-server/src/settings.rs:133:26
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tessus** [comment]
I am using the same config as in the guide I wrote on the forum. My server is atuin 18.3.0.

It could be possible that there's been a regression since then. I haven't tried git master server-side. I believe the issue is here: 

https://github.com/atuinsh/atuin/blob/main/crates/atuin-server/src/settings.rs#L68

I believe it should be `pub mail: Option<Mail>,` instead.

**Comment by pbek** [comment]
For now, I got over the error with:

```toml
[mail]
enable = false
```

**Comment by tessus** [comment]
Yep, setting a default value in the builder should work too (after line 109):

```rust
    .set_default("mail.enable", false)?
```

**Comment by pbek** [comment]
By the way, is this feature turned on on the official server? I registered an account today and never got an email.

**Comment by tessus** [comment]
Idk, this is a question for ellie...

**Comment by ellie** [comment]
@pbek it is enabled, however we do not email users for verification by default. We don't even verify that what users submit as an email _is_ an email.

Because of this, you can choose to verify or not verify. In the future, we will only support password recovery for verified emails, and if you choose not to do so you could lose access to your account.

It's still unreleased though, but if you're running a pre-release on the client as well as the server you can call `atuin account verify`.

**Comment by ellie** [comment]
Fixed :)

I appreciate you highlighting the issue, though you might have a better experience running a tagged release vs latest built image

**Comment by pbek** [comment]
Thank you very much for clarifying and for fixing!
And thank you very much for Atuin, I got reminded of it again in an article in https://www.linux-magazine.com/Issues/2024/281!

**Comment by pbek** [comment]
> though you might have a better experience running a tagged release vs latest built image

Was this a note for me? Do you mean in my docker setup? I didn't find any other tags than the commit hashes. I'd much prefere "latest". 😊

**Comment by ellie** [comment]
It was!

We don't ship `latest` intentionally. Releases may ship breaking changes, and following a moving tag will lead to your install not functioning. Self hosting is rarely just set and forget 😁 

I'd pick the latest stable release, tag that, and update whenever you want/need to

**Comment by pbek** [comment]
Ah, so there are other tags. I'll try to find them. Thank you!

(btw. do you ever sleep? 😂)

**Comment by ellie** [comment]
Not a problem!

> thank you very much for Atuin

Glad to hear you like it ☺️

> do you ever sleep?

Hahaha sometimes 🤣

**Comment by pbek** [comment]
> Hahaha sometimes 🤣

In between commits? 😁
